.mise.toml

```diff
@@ -1,2 +1,2 @@
 [env]
-_.file = '.env'
+_.file = '.env.sample'
```

Comment:
That's a sample file. During onboarding, you're supposed to copy its contents into a `.env` file, as we do in all our projects. Please revert.